packages/react-stand-in/src/createClassProxy.js

```diff
@@ -82,20 +85,23 @@ function createClassProxy(InitialComponent, proxyKey, wrapResult = identity) {
       // This is a Relay-style container constructor. We can't do the prototype-
       // style wrapping for this as we do elsewhere, so just we just pass it
       // through as-is.
-      if (isReactIndeterminateResult(result)) {
+      if (isReactComponentInstance(result)) {
         ProxyComponent = null
         return result
       }
 
       // Otherwise, it's a normal functional component. Build the real proxy
       // and use it going forward.
       ProxyComponent = proxyClassCreator(Component, postConstructionAction)
+
+      // We don't need to use safeDefineProperty here, as we control the entire
```

Comment:
I hope this is correct. Otherwise we should change this to use `safeDefineProperty` as well.